digitalocean/resource_digitalocean_monitor_alert.go

```diff
@@ -46,8 +46,10 @@ func resourceDigitalOceanMonitorAlert() *schema.Resource {
 					godo.LoadBalancerConnectionUtilizationPercent,
 					godo.LoadBalancerDropletHealth,
 					godo.LoadBalancerTLSUtilizationPercent,
-					godo.LoadBalancerIncreaseInHTTPErrorRatePercentage,
-					godo.LoadBalancerIncreaseInHTTPErrorRateCount,
+					godo.LoadBalancerIncreaseInHTTPErrorRatePercentage4xx,
```

Comment:
Hey @jrolheiser  do you mind also adding these strings to the .md file? like this? [https://github.com/digitalocean/terraform-provider-digitalocean/pull/912/files#diff-52b1fa155d23455f3f7e3302813f[…]4f4791ebb204709c44b8b28fcfR61-R62](https://github.com/digitalocean/terraform-provider-digitalocean/pull/912/files#diff-52b1fa155d23455f3f7e3302813f1767c14ee84f4791ebb204709c44b8b28fcfR61-R62)

Reply:
Done